hack/update-client-gen.sh

```diff
@@ -6,7 +6,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+SCRIPT_ROOT=$(dirname $(realpath "${BASH_SOURCE}"))/..
```

Comment:
Should we use `realpath` in the other scripts too....or is this one in particular a symlink sometimes?